.github/workflows/pull-request.yml

```diff
@@ -247,161 +260,26 @@ jobs:
-        run: |
-          timeout=120
-          echo "Starting development health check with ${timeout}s timeout"
-          while ! nc -z localhost 4321 && [ $timeout -gt 0 ]; do
-            sleep 1
-            timeout=$((timeout-1))
-            if [ $((timeout % 10)) -eq 0 ]; then
-              echo "Still waiting for development app to start... ${timeout}s remaining"
-            fi
-          done
-
-          if [ $timeout -eq 0 ]; then
-            echo "Timeout waiting for development application to start"
-            exit 1
-          fi
-          echo "Development app started successfully"
-
-      - name: Stop Development App
-        if: always()
-        run: |
-          if [ -f .pidfile_dev ]; then
-            kill "$(cat .pidfile_dev)"
-          fi
-
-  Docker-Start-Check:
-    name: Check if Talawa Admin app starts in Docker
-    runs-on: ubuntu-latest
-    needs: [Code-Quality-Checks, Test-Application]
-    if: github.actor != 'dependabot'
-    steps:
-      - name: Checkout the Repository
-        uses: actions/checkout@v4
-
-      - name: Set up Docker
-        uses: docker/setup-buildx-action@v3
-        with:
-          driver-opts: |
-            image=moby/buildkit:latest
-
-      - name: Build Docker image
-        run: |
-          set -e
-          echo "Building Docker image..."
-          docker build -t talawa-admin-app .
-          echo "Docker image built successfully"
-
-      - name: Run Docker Container
-        run: |
-          set -e
-          echo "Started Docker container..."
-          docker run -d --name talawa-admin-app-container -p 4321:4321 talawa-admin-app
-          echo "Docker container started successfully"
-
-      - name: Check if Talawa Admin App is running
-        run: |
-          timeout="${HEALTH_CHECK_TIMEOUT:-120}"
-          echo "Starting health check with ${timeout}s timeout"
-          while ! nc -z localhost 4321 && [ $timeout -gt 0 ]; do
-            sleep 1
-
-            timeout=$((timeout-1))
-            if [ $((timeout % 10)) -eq 0 ]; then
-              echo "Still waiting for app to start... ${timeout}s remaining"
-            fi
-          done
-
-          if [ $timeout -eq 0 ]; then
-            echo "Timeout waiting for application to start"
-            echo "Container logs:"
-            docker logs talawa-admin-app-container
-            exit 1
-          fi
-          echo "Port check passed, verifying health endpoint..."
-
-      - name: Stop Docker Container
-        if: always()
-        run: |
-          docker stop talawa-admin-app-container
-          docker rm talawa-admin-app-container
-
   Check-Target-Branch:
     if: ${{ github.actor != 'dependabot[bot]' }}
     name: Check Target Branch
     runs-on: ubuntu-latest
     steps:
       - name: Check if the target branch is develop
-        if: github.event.pull_request.base.ref != 'develop-postgres'
+        if: github.event.pull_request.base.ref != 'develop'
```

Comment:
@prayanshchh Another case of reverting a previous PR. You are working on the `develop-postgres` branch and then reverted it back to `develop`